h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -74,6 +75,58 @@ public void initActualParamValues() {
     //the training frame used for blending (from which predictions columns are computed)
     public Key<Frame> _blending;
 
+    public enum MetalearnerTransform {
+      NONE(null),
+      Logit(new MRTask() {
+        @Override
+        public void map(Chunk[] cs) {
+          double p, logit;
+          for (int c = 0; c < cs.length; c++) {
+            for (int i = 0; i < cs[c]._len; i++) {
+              p = cs[c].atd(i);
+              logit = Math.log(p/(1-p));
+              cs[c].set(i, Double.isInfinite(logit) ? Double.NaN : logit);  // Algos have troubles dealing with Inf
```

Comment:
are you sure it is okay to mutate the frame? are you changing the prediction frames?

Reply:
Thanks, good point!
No, it is not safe. The frame is created by taking one prediction column (vec) from each base model the predictions can be "CV predictions" in normal stacking mode so that shouldn't be safe. 

There are two uses of that MRTask - training and scoring. I think the mutation during scoring should be OK. During training it could modify cv predictions so it is not OK. 

I tried using `map(Chunk[], NewChunk[])` and `doAll(int numberOfOutputs, byte outputType, Frame fr).outputFrame(Key, String[], String[][])`, but somewhere under the hood `vecs` are deleted during second run - I guess SE specific logic for handling `levelOneFrame`, so I am looking more in to that.

Reply:
Good point, thanks! Sorry for my late reply - somehow I replied in "review" mode and I didn't finish the review and noticed it today...

I was modifying prediction frames and also cv predictions - which is not okay... I struggled with missing chunks during second run which I found out was due to reusing the same instance of `MRTask` 🤦 .

Now the goal is to have it in a state to work well enough to be able to benchmark it to find out if those transformations help SE be better.